plugins/auth-backend-module-oidc-provider/report.api.md

```diff
@@ -37,15 +38,8 @@ export type OidcAuthResult = {
 
 // @public
 export namespace oidcSignInResolvers {
-  const emailLocalPartMatchingUserEntityName: SignInResolverFactory<
-    unknown,
-    | {
-        allowedDomains?: string[] | undefined;
-      }
-    | undefined
-  >;
-  const emailMatchingUserEntityProfileEmail: SignInResolverFactory<
-    unknown,
```

Comment:
These are no longer needed as `module.ts` already includes `commonSignInResolvers`.

Reply:
Indeed! 👍 perhaps you could update the PR to only include this change?